src/SerializableLimitIterator.php

```diff
@@ -18,17 +18,13 @@ class SerializableLimitIterator extends LimitIterator implements Serializable, A
 {
     /**
      * Offset to first element
-     *
-     * @var int
      */
-    private $offset;
+    private int $offset;
```

Comment:
This is unrelated change.

Reply:
Sorry, it might have come after rebasing :/

Reply:
Now you removed the typehint.

Sorry for so much hassle for such a simple change